input.ini

```diff
@@ -9,10 +9,14 @@ work_dir = /path/to/work_dir
 imagery_dir = /path/to/imagery_dir
 # Raytheon corrected RPC file directory
 rpc_dir = /path/to/rpc_dir
+# Config file for VisSat
+aoi_config = /path/to/config.json
 
 [aoi]
 # The name of this AOI, a prefix to output files; required
 name = jacksonville
+# The UTM zone that this AOI is located in; required for VisSat
+utm = 17R
```

Comment:
Ideally we wouldn't need to specify this `utm` value here and it's either already contained in the VisSat AOI config, or is derivable from that information.